### PR TITLE
ArgoWF first main PR

### PR DESCRIPTION
**Pull request by jaxels10**
This is the initial PR into main for argoWF. I have added the following to the helm chart: 
Dependency on minio
Dependency on argowf
An example object bucket claim 
An example service account, rolebinding and role

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+version: 0.1.0
+
+dependencies:
+  - name: argo
+    version: "v0.16.8"
+    repository: "https://argoproj.github.io/argo-helm"
+  - name: minio
+    version: 8.0.9
+    repository: https://helm.min.io/
+    condition: minio.install
```
Comment (jacobcrawford) [inline chart/Chart.yaml:14]: We normally use "enabled" for conditionals on dependencies.

**Review by jacobcrawford — CHANGES_REQUESTED** [review]

File: chart/Chart.yaml (hunk 1 above)
Comment (skovbakke) [reply chart/Chart.yaml:14]: Or we could have a new variable `installMinIO`

**Review by skovbakke — CHANGES_REQUESTED** [review]
The Minio chart we reference is deprecated

File: chart/Chart.yaml (hunk 1 above)
Comment (jaxels10) [reply chart/Chart.yaml]: Removed minio

**Comment by jaxels10** [comment]
I have removed minio from the chart

**Review by skovbakke — APPROVED** [review]
Neat

**Review by jacobcrawford — APPROVED** [review]

**Pull request merged**